app/src/utils/hooks.js

```diff
@@ -5,14 +5,17 @@ function noop() {}
 
 export function useExternalContract(address, abi) {
   const api = useApi()
-  const [contract, setContract] = useState(null)
+  const canInstantiate = Boolean(api && address && abi)
```

Comment:
should this hooks file be inside this utils folder?

Reply:
We can probably move it to the hooks folder and call it `general-hooks` maybe?